src/libsystemd/sd-journal/journal-file.c

```diff
@@ -487,6 +487,8 @@ static bool warn_wrong_flags(const JournalFile *f, bool compatible) {
                         if (compatible) {
                                 if (flags & HEADER_COMPATIBLE_SEALED)
                                         strv[n++] = "sealed";
+                                if (flags & HEADER_COMPATIBLE_SEALED_CONTINUOUS)
```

Comment:
need to extend the strv[] size above from 6 to 7

Reply:
I don't think so. This method can be called with `compatible == true` or with `compatible == false`. There are 5 incompatible flags, and together with the `NULL`-terminator this requires 6 slots. The 2 compatible flags require 3 slots which is less.

Reply:
oh right